lib/download-main-artifact/index.mjs

```diff
@@ -8,24 +8,22 @@ const {
     REPO_OWNER = 'babel',
     REPO_NAME = 'babel',
     BRANCH = 'main',
-    WORKFLOW_NAME = 'CI',
+    WORKFLOW_NAME = 'ci.yml',
     ARTIFACT_NAME = 'test262-result',
     ARTIFACT_FILE = 'test262.tap',
 } = process.env;
 
 const octokit = new Octokit({ auth: GITHUB_TOKEN })
 
-
-const { workflow_runs } = await octokit.request('GET /repos/{owner}/{repo}/actions/runs?branch={branch}&per_page=100&name=test262', {
-    owner: REPO_OWNER,
-    repo: REPO_NAME,
-    branch: BRANCH
-}).then(handleErrors);
-
 let artifact;
-for (const workflow of workflow_runs) {
-    if (workflow.name !== WORKFLOW_NAME) continue;
-
+for (let page = 1;; page++) {
```

Comment:
Can we still keep a limit, just in case something goes wrong?

Reply:
Sure, we can keep 100. I am out of town now, could you commit the change? Thank you.